ci/pipelines/deploy-to-production.yml

```diff
@@ -989,6 +1055,40 @@ jobs:
     <<: *put_success_slack_notification_p1      
     <<: *put_failure_slack_notification              
 
+  - name: retag-carbon-relay-image-for-test-perf
+    plan:
+      - get: pay-ci
+      - get: carbon-relay-ecr-registry-prod
+        params:
+          format: oci
+        passed: [deploy-carbon-relay-to-prod]
+        trigger: true
+      - task: parse-perf-release-tag
+        file: pay-ci/ci/tasks/parse-perf-release-tag.yml
+        input_mapping:
+          ecr-repo: carbon-relay-ecr-registry-prod
+      - put: carbon-relay-ecr-registry-test
+        params:
+          image: carbon-relay-ecr-registry-prod/image.tar
+          additional_tags: parse-perf-release-tag/tag
+
+  - name: retag-stunnel-image-for-test-perf
+    plan:
+      - get: pay-ci
+      - get: stunnel-ecr-registry-prod
+        params:
+          format: oci
+        passed: [deploy-carbon-relay-to-prod]
+        trigger: true
+      - task: parse-perf-release-tag
+        file: pay-ci/ci/tasks/parse-perf-release-tag.yml
```

Comment:
I think we're going to have to do something different for egress, stunnel and carbon relay - the tags will need to be something like `10-egress-perf` (like we have `10-stunnel-release`).

Reply:
Yeah you're right